.github/workflows/macos.yml

```diff
@@ -0,0 +1,45 @@
+name: macOS latest
```

Comment:
Sweet, we should use this template for the other libraries too! https://github.com/ignition-tooling/action-ignition-ci/issues/25